gtk/Notify.cc

```diff
@@ -66,6 +68,19 @@ void get_capabilities_callback(Glib::RefPtr<Gio::AsyncResult>& res)
     }
 }
 
+void torrent_start_now(tr_session* session, int id)
+{
+    tr_variant top;
+    tr_variantInitDict(&top, 2);
+    tr_variantDictAddStr(&top, TR_KEY_method, "torrent-start-now");
+
+    auto args = tr_variantDictAddDict(&top, TR_KEY_arguments, 1);
+    auto ids = tr_variantDictAddList(args, TR_KEY_ids, 1);
+    tr_variantListAddInt(ids, id);
+    tr_rpc_request_exec_json(session, &top, nullptr, nullptr);
```

Comment:
I'd move this whole function to `Session`, or use `Session::exec()` (`core->exec()`) to avoid accessing `tr_session` directly. Could be done later as part of bigger refactoring, so not going to insist.

Reply:
Done, thanks.